### ISSUE TITLE
The CloudFormation template is invalid: Template format error: Outputs count 62 is greater than max allowed 60

### ISSUE THREAD

**Issue by hamecodedVia** [jagdish-176/serverless-nested-stack#5]
Having integrated the plugin, I now get a 60 cap instead of 200 cap

before:
`The CloudFormation template is invalid: Template format error: Number of resources, 214, is greater than maximum allowed, 200`

after:
`The CloudFormation template is invalid: Template format error: Outputs count 62 is greater than max allowed 60`

**Comment by dfloresgonz** [comment]
Did you get to solve this? I'm also hitting this error.

**Comment by znahas** [comment]
Yeah me too got this error...I think this plugin is unmaintained...switching to https://github.com/dougmoscrop/serverless-plugin-split-stacks
